common/lib/xmodule/xmodule/tabs.py

```diff
@@ -121,14 +133,8 @@ def __setitem__(self, key, value):
 
         Note: the 'type' member can be 'get', but not 'set'.
```

Comment:
Good catch. I needed to add more properties, and it seemed much simpler to just allow them all to be supported. I wonder why it matters that you can't set type. I'll check, and if so I'll add an assertion to make sure it can't be changed. Of course, I'll then have to write tests just for this one case...